xtas/tasks/_polarity.py

```diff
@@ -68,7 +68,7 @@ def train(param_search=False):
                                         strip_accents='unicode'),
                         LogisticRegression(random_state=623, C=5000))
 
-    if param_search:
```

Comment:
Maybe. This is really a quick fix to the problem that a parameter sweep takes extremely long.